README.md

```diff
@@ -4,6 +4,42 @@ This project does not build currently. It is in progress.
 
 Below is work in progress
 
+## Shipkit workflow test drive in 4 simple steps
+
+1. **Fork & clone**: fork this repo and clone it to your computer
+2. **Dummy change**:  check out new branch, make some dummy change, commit and push to remote repo.
+You can just copy paste below to your terminal:
+
+```
+git checkout -b shipkit-testing
+echo "Shipkit testing" >> README.md
+git commit README.md -m "Shipkit testing"
+git push --set-upstream origin shipkit-testing
+```
+
+3. **Pull request**: in GitHub, open PR with the dummy change pushed in the previous step.
+**Important**: set the "base fork" to your own fork because GitHub by default suggests upstream repo.
+Upstream repo is no good because you don't have permissions and you cannot merge immediately.
```

Comment:
In GitHub Web UI it's called "base fork" or "base repository". This instruction step is for using GitHub Web and I will accompany it with a sweet screenshot :)

I'm moving on merging if that's ok, we can discuss / update the naming in the next PR.